### DISCUSSION TITLE
Twitter Space: GitHub Discussions

### DISCUSSION THREAD

**Discussion by BekahHW** [deepgram/community#28] [General help]
## GitHub Discussions Twitter Space Follow-up
If you have questions, comments, or thoughts on GitHub discussions and we didn't get to them in the space, drop them here!

![Twitter Space image card with date Sept 23 at 12p PT with Bekah Nad Lindsey Bieda's headshots](https://user-images.githubusercontent.com/34313413/192001858-dfbef4b4-58d4-41c0-86eb-31c15f78ce2a.png)

**Comment by JesseRWeigel** [comment]
Great discussion! I just started using discussions in one of my repos. I think the poll feature will be very useful.
